pkgs/by-name/th/thorium/package.nix

```diff
@@ -0,0 +1,40 @@
+{
+  stdenv,
+  fetchurl,
+  undmg,
+  lib,
+}:
+stdenv.mkDerivation (finalAttrs: rec {
```

Comment:
`finalAttrs` and `rec` are not meant to be used together.

```suggestion
stdenv.mkDerivation (finalAttrs: {
```

Reply:
@quinneden Is this marked as resolved by accident? There seems no further changes uploaded nor comments explaining the decision.

Reply:
> @quinneden Is this marked as resolved by accident? There seems no further changes uploaded nor comments explaining the decision.

Yes, my apologies.